frontend/src/components/TradeBox/index.js

```diff
@@ -1438,6 +1442,29 @@ class TradeBox extends Component {
             </b>{' '}
             {' ' + this.stepXofY()}
           </Typography>
+          <Grid item>
```

Comment:
If placed as the first item of `<Grid container ...>` and with `style={{position:'fixed', right:'1em'}}` the Switch would probably render on the top right corner of the TradeBox. No need to fix as this work would have to be re-done next week on the new TradeBox anyway.